docs/Privacy/Privacy.md

```diff
@@ -1,92 +1,93 @@
-transaction, the sending party, or the list of participating addresses.
-The sending actor is specified in the `privateFrom` attribute of the 
-transaction. The receiving parties are specified in the `privateFor` 
-attribute of the transaction. See [eea_sendrawtransaction](../Reference/JSON-RPC-API-Methods.md#eea_sendRawTransaction)
-for implementation details.
-
-### Important Concepts
-- **Precompiled Contract**: A smart contract compiled from its source
- language to EVM bytecode and stored by an Ethereum node for later
- execution.
-- **Privacy**: It is “The right of individuals to control or influence
- what information related to them may be collected and stored and by
- whom and to whom that information may be disclosed.” For the purposes
- of this Specification, this right of privacy also applies to
- organizations to the extent permitted by law.
-- **Private State**: Data that is not shared in the clear in the
- globally replicated state tree. This data can represent bilateral or
- multilateral arrangements between counterparties, for example in
- private transactions.
-- **Private Transaction**: A transaction where some information about
- the transaction, such as the payload data, or the sender and
- recipient, is only available to the subset of network actors who are
- parties to that transaction.
-- **Private Transaction Manager**: A subsystem of an Enterprise Ethereum
-  system for implementing privacy known as an Enclave.
-  e.g. [Orion](https://github.com/PegaSysEng/orion).
-- **Actor**: The public key of someone involved in the private transaction (e.g. a `privateFor` actor).
-- **Privacy group id**: A unique identifier associated with the set of actors
- that are involved in private transaction.
-
-
-### How Private Transaction is Processed
-
-1. A private transaction is sent to the [eea_sendrawtransaction](../Reference/JSON-RPC-API-Methods.md#eea_sendRawTransaction)
-endpoint, using the `privateFor` attribute to specify the list of actors,
-The `privateFrom` attribute specifies the sending actor, and `restriction`
-attribute is set to *restricted*.
- 
-2. The JSON RPC endpoint hands off the private transaction to a
-Private Transaction Handler.
+# Privacy
+
+Privacy in Pantheon refers to the ability to keep transactions private between the 
+involved parties. Other parties have no access to the transaction content, the sending party, or the 
+list of participating parties. 
+
+This page describes what private transactions are and how they are processed in Pantheon. Other pages describe:
+
+* How to configure Pantheon and the Private Transaction Manager (for example, Orion) to send private transactions
+* Creating and sending private transactions 
+
+## Private Transactions
+
+Private transactions are defined as *restricted* or *unrestricted*:  
+
+* In restricted private transactions the payload of the private transaction is stored only in the nodes
+participating in the transaction. 
+
```

Comment:
@rojotek can you confirm that with Orion only the participating nodes receive the payload. Might be worth calling out? 

 `In restricted private transactions, when using Orion, the payload of the private transaction is stored and only available to the participating nodes`

Reply:
Updated to reflect that private transactions are only received by participating nodes